hms-lambda-handler/src/main/java/com/amazonaws/athena/hms/S3ASchemeUpdater.java

```diff
@@ -0,0 +1,37 @@
+/*-
+ * #%L
+ * hms-lambda-handler
+ * %%
+ * Copyright (C) 2019 - 2022 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.hms;
+
+import org.apache.hadoop.hive.metastore.api.Table;
+
+public class S3ASchemeUpdater {
+    private static final String S3_SCHEME = "s3:";
```

Comment:
Is there a constant already existing in the package that we can reuse?

Reply:
Thanks CY. Currently there is no constant can be reused.